config/hubs/pangeo-hubs.cluster.yaml

```diff
@@ -0,0 +1,73 @@
+name: pangeo-hubs
+provider: gcp
+gcp:
+  key: secrets/pangeo-hubs.json
+  project: pangeo-integration-te-3eea
+  cluster: pangeo-hubs-cluster
+  zone: us-central1-b
+hubs:
+  - name: staging
+    domain: staging.pangeo.2i2c.cloud
+    template: daskhub
+    auth0:
+      connection: github
+    config: &stagingConfig
+      basehub:
+        jupyterhub:
+          proxy:
+            https:
+              enabled: false
+          custom:
+            homepage:
+              templateVars:
+                org:
+                  name: Pangeo
+                  url: https://pangeo.io
+                  logo_url: "https://coessing.files.wordpress.com/2016/08/ghana-logo-21.png?w=262&h=376&zoom=2"
+                designed_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                operated_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                funded_by:
+                  name: Pangeo
+                  url: https://pangeo.io
+          hub:
+            config:
+              Authenticator:
+                admin_users:
+                  - jhamman
+                  - rabernat
+                  - TomAugspurger
+                  - scottyhq
+                  - sgibson91
+                  - yuvipanda
+                  - damianavila
+              GitHubOAuthenticator:
+                allowed_organizations:
+                  # Allow access to any member of the us-central1-b-gcp team
```

Comment:
Let's make a note that this doesn't have any effect right now?

Reply:
Yeah, I'm having trouble with GitHub and my local copy because I've removed this and pushed, but the commit isn't showing up here. Might close this and start afresh (silly GitHub outage!)